### ISSUE TITLE
Temperature measurements

### ISSUE THREAD

**Issue by JorisVincent** [BrainardLab/OneLightToolbox#28]
At some point in the near future, it might be useful to integrate the temperature measurements better into our workflow. Right now, some functions take temperature measurements (e.g., most of the `OLCalibrateOOC`-stack), while others do not (notable exception: `OLWarmupOOC` does **not** take temperature measurements). There is some infrastructure for analyzing temperature measurements, but it's not straightforward how and where this is used.

It would probably also be good to integrate temperature measurements and analysis in our experimental pipeline, e.g., in the corrections and validations. While I don't see this immediately solving some problem, it would be good to have this information (especially now that we're diving deeper into how components in and factors of the boxes interact).

**Comment by JorisVincent** [comment]
Obviously no rush on this, but I noticed that `OLWarmupOOC` does not take temperature measurements, and wanted to make a note of this before it got lost from memory.

**Comment by DavidBrainard** [comment]
I totally agree that we want to make the temp measurements work routinely with everything we do.  I wonder if it would be worth having a setting where OLTakeMeasurement or whatever that routine is called takes a temp measurement, controlled by a key/value pair.  If the temp were returned as part of the measurement structure, then we'd probably have it around.  We definitely want the warmup routine to take and store these measurements.
